### PR TITLE
 adding sonaqube dashboard and sonaqube core into our dashboards

### PR DESCRIPTION
**Pull request by sherrif10**

### CODE REVIEW

**Comment by sherrif10** [comment]
@kdaud  is this sufficient

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Install & Upgrade Tests |
  [![Platform Advanced Installation](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-install-advanced.yml/badge.svg?branch=master)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-install-advanced.yml) 
  [![Platform Postgres Installation](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-install-postgres.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-install-postgres.yml) 
  [![Platform Upgrade](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-upgrade.yml/badge.svg?branch=master)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-upgrade.yml) |
+[openmrs core sonaqube](https://ci.openmrs.org/browse/SON-OPENMRSCOREMASTER) ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/LU-LU) 
+[openmrs sonaqube dashboard](https://sonar.openmrs.org/dashboard?id=org.openmrs%3Aopenmrs) ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/LU-LU) |
```
Comment (kdaud) [inline README.md:19]: do we really need two budges for the sonaqube?

Comment (sherrif10) [reply README.md:19]: Yes we need them, these are not badges, one is referring to openmrs  core sonaqube ci and  and another one is for sonaqube dashboardPage

Comment (kdaud) [reply README.md:19]: do we need to have sonaqube dashboard here? isn't openmrs core sonaqube ci good enough?

**Comment by sherrif10** [comment]
It's doesn't show sonaqube dashboard but rather shows openmrs Sona builds,
the idea would be have them both

**Comment by kdaud** [comment]
all the two guys are tracking the same stuff via `https://ci.openmrs.org/plugins/servlet/wittified/build-status/LU-LU`, isn't one of them good enough?

**Comment by sherrif10** [comment]
Not really this is neither tracking any code base not any functionality, it was asked by christine to add the link to sonaqube web such that we can easily track vulnabilities via web, there is no need of adding it in the same column.

**Comment by kdaud** [comment]
Screenshot from 2021-11-17 19-37-41
I would expect to have sonaqube on the same raw with TRUNK, REST API, and FIHR API not under `Install and Upgrade tests`

**Comment by sherrif10** [comment]
Trunk is tracking same thing as openmrs core sonaqube. Have u recognized
that

**Comment by sherrif10** [comment]
There is no specific place its supposed to be in, we can put it under install &upgrade  tests.

**Comment by sherrif10** [comment]
@kdaud i guess this is fine to get merged.

**Pull request merged**